axon-framework/events/event-processors/README.md

```diff
@@ -373,6 +379,74 @@ If either of these thresholds is exceeded, the queue will throw a `DeadLetterQue
 This exception means the processing group will stop processing new events altogether.
 Thus, the processing group moves back to the behavior described at the start of the [Error Handling](#error-handling) section.
 
+#### Configuring a sequenced Dead-Letter Queue Provider
+
+In order to make it easier to use a dead-letter queue on multiple processing groups, it's possible to set a dead-letter queue provider.
+This is a function that takes a processing group, and returns either `null`, meaning it will not be configured using a dead-letter queue, or a function that takes `Configuration` and returns a new dead letter queue.
```

Comment:
```suggestion
The provider is a function that takes a processing group, and returns either `null`, meaning it will not be configured using a dead-letter queue, or a function that takes the `Configuration` and returns a new dead-letter queue.
```